examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx

```diff
@@ -1,15 +1,15 @@
-import '../styles/globals.css';
+// import '../styles/globals.css';
```

Comment:
Should this be deleted if it's commented out?

Reply:
Good catch.